cadcutils/cadcutils/old_get_cert.py

```diff
@@ -0,0 +1,129 @@
+
+    dirname = os.path.dirname(cert_filename)
+    try:
+        os.makedirs(dirname)
+    except OSError as oex:
+        if os.path.isdir(dirname):
+            pass
+        elif oex.errno == 20 or oex.errno == 17:
+            sys.stderr.write("%s : %s \n" % (str(oex), dirname))
+            sys.stderr.write("Expected %s to be a directory.\n" % dirname)
+            sys.exit(oex.errno)
+        else:
+            raise oex
+    username, passwd = get_user_password(cert_server)
+
+    url = "http://{0}/{1}".format(cert_server, cert_endpoint)
+    resp = requests.get(url, params=kwargs, auth=(username, passwd))
+    if resp.status_code != 200:
+        raise OSError(resp.status_code, html2text.html2text(resp.content))
+    with open(cert_filename, 'w') as w:
+        w.write(resp.content)
+    return resp
+
+
+def get_user_password(realm):
+    """"Getting the username/password for realm from .netrc filie.
+
+    :param realm: the server realm this user/password combination is for
+    :ptype realm: str
+    """
+    if os.access(os.path.join(os.environ.get('HOME', '/'), ".netrc"), os.R_OK):
+        auth = netrc.netrc().authenticators(realm)
+    else:
+        auth = False
+    if not auth:
+        sys.stdout.write("{0} Username: ".format(realm))
+        username = sys.stdin.readline().strip('\n')
+        password = getpass.getpass().strip('\n')
+    else:
+        username = auth[0]
+        password = auth[2]
+    return username, password
+
+
+def _main():
+
+    def _signal_handler(signal, frame):
+        sys.stderr.write("\n")
+        sys.exit(-1)
+
+    signal.signal(signal.SIGINT, _signal_handler)
+
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+                                     description=("Retrieve a security certificate for interation with VOSpace. "
+                                                  "Certificate will be valid for daysValid and stored as local file "
+                                                  "cert_filename. First looks for an entry in the users .netrc "
+                                                  "matching the realm {0}, the user is prompted for a username and "
+                                                  "password if no entry is found.".format(CERT_SERVER)))
+
+    parser.add_argument('--daysValid', type=int, default=10, help='Number of days the cetificate should be valid.')
+    parser.add_argument('--cert-filename',
+                        default=os.path.join(os.getenv('HOME', '/tmp'), '.ssl/cadcproxy.pem'),
+                        help="Filesysm location to store the proxy certifcate.")
+    parser.add_argument('--cert-server',
+                        default=CERT_SERVER,
+                        help="Certificate server network address.")
+
+    args = parser.parse_args()
+
+    retry = True
+    print("The getCert application is being deprecated. Please start using the cadc-get-cert application instead.")
```

Comment:
Likely this should be handled with as

DeprecationWarning(blah blah).

Then it can be suppressed and users only see once in execute cycle.